lib/components/artist/shows/large_image_show.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { View, Text, StyleSheet, Dimensions } = React;
+
+import Headline from '../../text/headline'
+import SerifText from '../../text/serif'
+import OpaqueImageView from '../../opaque_image_view'
+import ShowMetadata from './metadata'
+
+const windowDimensions = Dimensions.get('window');
+const imageWidth = windowDimensions.width > 700 ? (windowDimensions.width - 85)/2 : (windowDimensions.width - 40);
```

Comment:
I'm having some trouble here. The iPad Past Shows spec has 3 columns, and if I change the 2 in `(windowDimensions.width - 85)/2` to a 3, this image will be rendered nicely in a `LargeList`. The problem is that the `LargeList` doesn't know if it's showing past shows or current shows. Is there a clever way to deal with this? The brute-force approach is to make a whole Past Shows list component and `medium_image_show`.